src/wallet/wallet.h

```diff
@@ -198,7 +196,8 @@ class CMerkleTx : public CTransaction
     int GetDepthInMainChain() const { const CBlockIndex *pindexRet; return GetDepthInMainChain(pindexRet); }
     bool IsInMainChain() const { const CBlockIndex *pindexRet; return GetDepthInMainChainINTERNAL(pindexRet) > 0; }
     int GetBlocksToMaturity() const;
-    bool AcceptToMemoryPool(bool fLimitFree=true, bool fRejectAbsurdFee=true);
+    /** Pass this transaction to the mempool. Fails if absolute fee exceeds maxTxFee. */
+    bool AcceptToMemoryPool(bool fLimitFree=true, CAmount nAbsurdFee=maxTxFee);
```

Comment:
Why not default to the constant DEFAULT_TRANSACTION_MAXFEE instead of the global maxTxFee ?